README.md

```diff
@@ -1,4 +1,4 @@
-# th2-conn-dirty-fix (1.1.0)
+# th2-conn-dirty-fix (1.2.0)
```

Comment:
Please follow the https://exactpro.atlassian.net/wiki/spaces/TH2/pages/127860737/Versioning document and update maintenance version after bug fix